scaling_performance/optimizing_storage.adoc

```diff
@@ -11,25 +11,185 @@
 
 toc::[]
 
-== Optimizing Storage
+This topic provides information for optimizing {product-title} storage.
+
+[[general-storage-guidelines]]
+== General Storage Guidelines
+
+The following table lists the persistent storage technology that {product-title} can be used with.
```

Comment:
"The following table lists the available persistent storage options for {product-title}."

Reply:
Technology (or technologies) is a appropriate here.  Most of the technologies are only supported buy purchasing them from a 3rd party.

Reply:
Thanks @lpsantil so then:

"The following table lists the available persistent storage technologies for {product-title}."